packages/react/slot/src/Slot.tsx

```diff
@@ -63,7 +63,7 @@ const SlotClone = React.forwardRef<any, SlotCloneProps>((props, forwardedRef) =>
   if (React.isValidElement(children)) {
     return React.cloneElement(children, {
       ...mergeProps(slotProps, children.props),
-      ref: composeRefs(forwardedRef, (children as any).ref),
+      ref: forwardedRef ? composeRefs(forwardedRef, (children as any).ref) : (children as any).ref,
```

Comment:
could also just do this tbh (so it isn't overriding `ref` if it doesn't need to) but lmk which you'd prefer :)

![Screenshot 2023-04-27 at 16 21 20](https://user-images.githubusercontent.com/175330/234909232-cf03da0b-d498-4fd4-9a9a-a05c310776a0.png)

Reply:
Wouldn't that one be missing `children.ref` then?

Reply:
I think I prefer the explicit way you have written though either way.